### DISCUSSION TITLE
How to keep n plot running all the time

### DISCUSSION THREAD

**Discussion by tomuGo** [Chia-Network/chia-blockchain#5396] [Q&A]
when i add plot to queue, Wait until the end of parallel plot,but only  one plot running .i want keep n plot running all the time,not 1.thinks

**Comment by OversizedWizard** [comment]
If you use ver1.1.2 you can create for example 4 plots in parallel, then just add more plots to the que.

Then when one plot finished, it would pick up one from the que and start.
This would happen for all the 4 plots I had parallel. So I always had 4 plot going.

Now with version 1.1.4 and 1.1.5, you can start lots of parallel plots, add plots to qué, but when one plot has finished, it will pick the next from the que, but when the other parallel plotting finishes, they don't pick another from the que, so your left with just one plot plotting.
